tests/core/audit_test.py

```diff
@@ -807,25 +807,33 @@ def test_basic(self, mock_printer):
         ):
             self.run_logic()
 
-        assert uncolor(mock_printer.message) == textwrap.dedent("""
-            Secret:      1 of 2
-            Filename:    filenameA
-            Secret Type: Private Key
-            ----------
-            10:a
-            11:b
-            12:c
-            13:d
-            14:e
-            15:-----BEGIN PRIVATE KEY-----
-            16:e
-            17:d
-            18:c
-            19:b
-            20:a
-            ----------
+        expected_message = (
+            'Secret:      1 of 2'
+            '\nFilename:    filenameA'
+            '\nSecret Type: Private Key'
+            '\n----------'
+            '\n10:a'
+            '\n11:b'
+            '\n12:c'
+            '\n13:d'
+            '\n14:e'
+            '\n15:-----BEGIN PRIVATE KEY-----'
+            '\n16:e'
+            '\n17:d'
+            '\n18:c'
+            '\n19:b'
+            '\n20:a'
+            '\n----------'
+            '\nA potential secret was detected in this code.'
+            ' If so, you should select "yes" below to mark it'
+            ' as an actual secret, and remediate it.'
+            '\nOnce the secret has been removed from the file,'
+            ' and another scan has been run,'
+            ' its entry will be removed from the baseline file.'
+            '\n----------\n'
+        )
```

Comment:
Multi-line strings can't be used in this case because the additional auditing text causes the maximum line length to be exceeded, failing the pre-commit flake8 check; max chars per line = 100, w/ this change a line is ~150 chars;

Reply:
Extract the common part into some global static variables?

Reply:
You're right, fixed here: https://github.com/IBM/detect-secrets/pull/43/commits/d306df3580ceea45ad14a1b2754e0b86d29d5473